package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@gnosis.pm/dex-contracts",
-  "version": "0.1.0",
+  "version": "0.1.3",
```

Comment:
I guess there were some version bumps in between.

Reply:
Let's coordinate our versioning with the frontend. It would be nice if e.g. from the second digit we can infer compatibility (e.g. v 0.2.x frontend works with 0.2.y services).